plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/security/KafkaEndpointIdentificationAlgorithm.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.security;
+
+import static java.util.Locale.ENGLISH;
+
+public enum KafkaEndpointIdentificationAlgorithm
+{
+    HTTPS("https"),
+    DISABLED("disabled");
```

Comment:
Should it be "disabled" or "". Do we have test coverage for that?

Reply:
Yes you are right. I'll change it to empty "". I modified `TestKafkaSecurityConfig#testExplicitPropertyMappings` to cover `DISABLED`.